src/poliastro/constants.py

```diff
@@ -0,0 +1,111 @@
+"""Astronomical and physics constants in SI units.
+
+This module complements constants defined in `astropy.constants`
+
+Unless otherwise specified, gravitational and mass parameters were obtained from:
```

Comment:
Also, if some of the masses are of the whole planetary system, we should say so.